src/Tribe/I18n.php

```diff
@@ -190,13 +253,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ($flags & static::COMPILE_INPUT) {
+						$strings[ $key ][] = __( $value );
+					}
+					if ($flags & static::COMPILE_STRTOLOWER) {
+						$strings[ $key ][] = __( strtolower( $value ) );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "strtolower( $value )".

<sub>hash: cebf7d378d4a94ea0f9f4a9b51e2e3db</sub>